docs/config.ini

```diff
@@ -0,0 +1,77 @@
+[project]
+
+# The project name, used as the filename of the package and the PDF file. For
+# example, if set to d2l-book, then will build d2l-book.zip and d2l-book.pdf
+name = autogluon
+
+# Book title. It will be displayed on the top-right of the HTML page and the
+# front page of the PDF file
+title = AutoGluon Documentation
+
+author = AutoGluon contributors
+
+copyright = 2022, All developers. Licensed under the Apache License, Version 2.0.
+
+release = 0.6.2
```

Comment:
Do we follow a different versioning mechanism from cloud module or we want to be always consistent with AG?